static/css/dashboard/domains.css

```
@@ -0,0 +1,453 @@
+    font-size: 13px;
+}
+
+.cascade-option-desc.warning {
+    color: var(--color-danger);
+}
+
+.fade-in {
+    animation: fadeIn 0.18s ease-in;
+}
+
+@keyframes fadeIn {
```

Stylelint (17.11.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)